test/cucumber/steps/ReportsTester.groovy

```diff
@@ -0,0 +1,51 @@
+/**
+ * Created by Milena Carneiro on 28/04/2016.
```

Comment:
(OK) - Test behavior should strictly conform to scenario semantics
(OK) - Avoid ambiguities due to similar step sentences
(OK) - Do not duplicate test code
(OK) - Test should clean up environment at the end
(OK) - Do not mix GUI and controller tests
(OK) - Tests should be platform and language independent